tests/clients/test_dapr_grpc_client.py

```diff
@@ -397,7 +397,7 @@ def test_wait_timeout(self):
         dapr = DaprGrpcClient(f'localhost:{port}')
         with self.assertRaises(Exception) as context:
             dapr.wait(0.1)
-        self.assertTrue('Connection refused' in str(context.exception))
+        self.assertTrue(type(context.exception) is socket.timeout)
```

Comment:
This change causes CI failures.

Reply:
It was failing locally, so I figured: Let's make it work again and see how it goes on CI. Is there a difference between Windows and Linux here?